maven_iuvs/echelle.py

```diff
@@ -1779,6 +1850,25 @@ def add_in_quadrature(uncertainties, light_fits, integration=0):
     return total_uncert
 
 
+def ran_DN_uncertainty(light_fits, dark_subtracted_and_cleaned_data):
```

Comment:
I think this uncertainty calculation is applicable to all modes, not just the echelle, right? If so, I would move this to the instrument submodule

Reply:
Fixed by [c18a68f](https://github.com/lasp/maven_iuvs/pull/66/commits/c18a68f35f885e8be9358442b4cb9cc0f8386452) and [fbf76a9](https://github.com/lasp/maven_iuvs/pull/66/commits/fbf76a9c1b5b8d4f6f4ac91b74360d9bc2579ef6)